docgen/generate-docs.js

```diff
@@ -103,6 +107,7 @@ function renameFiles() {
  */
 function fixLinks(file) {
   return fs.readFile(file, 'utf8').then(data => {
+    data = addTypeAliasLinks(data);
```

Comment:
I see we are doing this for all files, but not all the files have one of the three links we'd like to make external so we're doing a lot of extra work per each file. If it's easy to know which files need this adjustment, I would suggest adding this check to only those files. However, if you feel this will make the code harder to read, or if you don't have a clear idea of which files will need to be modified, feel free to leave as is.